File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int numP = 0;
        int numY = 0;

        for (char ch : s.toCharArray()) {
            if (Character.toLowerCase(ch) == 'p') {
                numP++;
            }

            if (Character.toLowerCase(ch) == 'y') {
                numY++;
            }
        }

        if (numP != numY) {
            answer = false;
        }

        // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
        System.out.println("Hello Java");

        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = false;
    
    let pCnt = 0;
    let yCnt = 0;

    for(let i of s.toLowerCase()){
        if(i == 'p'){
            pCnt++;
        }
        if(i == 'y'){
            yCnt++;
        }
    }
        if(pCnt == yCnt){
            answer = true;
        }
        if(pCnt == 0 && yCnt == 0){
            answer = true;
        }

    return answer;
}